R/classicProcess.R

```diff
@@ -1926,6 +1928,54 @@ ClassicProcess <- function(jaspResults, dataset = NULL, options) {
   }
 }
 
+
+.procRsquared <- function(options, modelsContainer) {
+  # adapted from .semRsquared() from jaspSem/R/sem.R (one table for several models)
+  if (!options[["rSquared"]] || !is.null(modelsContainer[["rsquared"]])) return()
+  
+  # retrieve model objects
+  procResults <- lapply(options[["processModels"]], function(mod) modelsContainer[[mod[["name"]]]][["fittedModel"]]$object)
+  modelNames <- sapply(options[["processModels"]], function(mod) mod[["name"]])
+  
+  # init table
+  tabr2 <- createJaspTable(gettext("R-Squared"))
+  tabr2$addColumnInfo(name = "__var__", title = "", type = "string")
+  for (i in seq_along(options[["processModels"]])) {
+    tabr2$addColumnInfo(name = paste0("rsq_", i), title = modelNames[i],
+                        overtitle = "R\u00B2", type = "number")
+  }
+  
+  tabr2$dependOn(c("rSquared", "processModels"))
```

Comment:
```suggestion
  tabr2$dependOn(c(.procGetDependencies(), "rSquared", "processModels"))
```

Here should also be the general dependencies that affect all process models, such as the estimator or variable standardization.